mod/beacon/validator/types.go

```diff
@@ -201,6 +210,16 @@ type PayloadBuilder[BeaconStateT, ExecutionPayloadT any] interface {
 	) (engineprimitives.BuiltExecutionPayloadEnv[ExecutionPayloadT], error)
 }
 
+// SlotData represents the slot data interface.
+type SlotData[AttestationDataT, SlashingInfoT any] interface {
+	// GetSlot returns the slot of the incoming slot.
+	GetSlot() math.Slot
+	// GetAttestationData returns the attestation data of the incoming slot.
```

Comment:
yeah these descriptions are technically incorrect right, its sorta the data for this slot, but its also sort of not